ow_lander/urdf/lander.xacro

```diff
@@ -844,7 +844,7 @@
     <axis
       xyz="0 0 -1" />
     <dynamics damping="50.0" />
-    <limit effort="80.0" velocity="0.2" />
+    <limit effort="80.0" velocity="0.2" lower="-3.14" upper="3.14" />

```

Comment:
We discussed pan a long time ago and settled on something greater than pi (I think it was 3.2), so if there's any kind of oscillation in the controller the joint won't need to wrap all the way around.

3.14 is actually a little less than pi and would leave a small gap through which you cannot pan. The same consideration should be given to other joints where these limits are used.

Reply:
The pan range we decided on a while back is in fact [-3.2, 3.2] .

Reply:
Done!

Reply:
This should also be done to other joints that currently use [-3.14, 3.14]. Although, j_prox_pitch could probably be tightened up to use a smaller range.

Reply:
Do we want the range to be -3.2 to 3.2 for all the joints that are free to rotate ?

Reply:
Seems like a reasonable guess. It's impossible to know how much wrap-around would be designed into a joint for a real mission.

Reply:
ok changed the defaults to 3.2